deploy/sre-authorization/01-osd-sre-admin.ClusterRole.yaml

```diff
@@ -135,3 +135,19 @@ rules:
   - certificatesigningrequests/approval
   verbs:
   - update
+# SRE can download backups, review logs, and create/delete backups/restores
+- apiGroups:
+  - velero.io
+  resources:
+  - backups
+  - restores
```

Comment:
```suggestion
```

We should not allow restore by default.  It's not a typical use case for SRE, we have only ever restored from backup once in v3.  As the configuration stands right now my understanding is restore would not work.  There are openshift-isms that need to be addressed.  We'll be looking to what the app migration team is doing with restore.  As a fallback, SRE can elevate privileges to do a restore if required.

Reply:
I'm fine with removing restore at this time.